### PR TITLE
Add user management

### PR DESCRIPTION
**Pull request by xiaoxinla**
1. Add change password to all users
2. Hide user management and cluster management for regular users
3. Add add-user, delete-user and reset-password functions for admins

### CODE REVIEW

File: webportal/src/app/user/user-login/user-login-nav.component.ejs [hunk 1]
```diff
@@ -5,6 +5,7 @@
       <span class="hidden-xs">Welcome, <%= cookies.get("user") %></span>
     </a>
     <ul class="dropdown-menu">
+			<li><a href="/useraccount.html">Account</a><li>
```
Comment (abuccts) [inline webportal/src/app/user/user-login/user-login-nav.component.ejs]: indent

File: webportal/src/app/layout/layout.component.ejs [hunk 2]
```diff
@@ -34,11 +34,31 @@
     <aside class="main-sidebar">
       <section class="sidebar">
         <ul class="sidebar-menu" data-widget="tree">
-          <li id="sidebar-menu--user-management">
-            <a href="/register.html">
+					<li class="treeview" id="sidebar-menu--user">
```
Comment (abuccts) [inline webportal/src/app/layout/layout.component.ejs]: indent

File: webportal/config/webpack.common.js [hunk 3]
```diff
@@ -44,6 +44,10 @@ const config = {
     layout: './src/app/layout/layout.component.js',
     register: './src/app/user/user-register/user-register.component.js',
     login: './src/app/user/user-login/user-login.component.js',
+    useraccount: './src/app/user/user-account/user-account.component.js',
+    adduser: './src/app/user/add-user/add-user.component.js',
```
Comment (abuccts) [inline webportal/config/webpack.common.js]: duplicate with `register`?

**Comment by fanyangCS** [comment]
please close this PR. breaking this PR into smaller ones and resubmit them

**Pull request closed without merging**